PWGCF/Flow/Tasks/FlowPbPbpikp.cxx

```diff
@@ -169,6 +170,38 @@ struct GfwPidflow {
     fGFW->CreateRegions();
   }
 
+  enum Particles {
+    PIONS,
+    KAONS,
+    PROTONS
+  };
+
+  enum Particles pion = PIONS;
+  enum Particles kaon = KAONS;
+  enum Particles proton = PROTONS;
```

Comment:
These variables are not actually needed 
The comparisons can be done directly with the `enum` named constants